From 39b0794e1572b5ab4dbf2db975ee5d3afc3d3b76 Mon Sep 17 00:00:00 2001
From: Nathan Rossi <nathan.rossi@xilinx.com>
Date: Fri, 7 Jun 2013 14:08:06 +1000
Subject: [PATCH] fdisk: add support for the MicroBlaze architecture

Add support to fdisk for the MicroBlaze architecture.

Signed-off-by: Nathan Rossi <nathan.rossi@xilinx.com>
Signed-off-by: Stephan Linz <linz@li-pro.net>

Conflicts:
	fdisk/fdiskbsdlabel.h
---
 fdisk/fdiskbsdlabel.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/fdisk/fdiskbsdlabel.h b/fdisk/fdiskbsdlabel.h
index e58e2ff..7536077 100644
--- a/fdisk/fdiskbsdlabel.h
+++ b/fdisk/fdiskbsdlabel.h
@@ -48,7 +48,8 @@
 
 #if defined (__i386__) || defined (__sparc__) || defined (__arm__) || \
     defined (__mips__) || defined (__s390__) || defined (__sh__) || \
-    defined(__x86_64__) || defined (__avr32__) || defined(__cris__)
+    defined (__x86_64__) || defined (__avr32__) || defined(__cris__) || \
+    defined (__microblaze__)
 #define BSD_LABELSECTOR   1
 #define BSD_LABELOFFSET   0
 #elif defined (__alpha__) || defined (__powerpc__) || defined (__ia64__) || defined (__hppa__)
-- 
1.8.3.4

